libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/SalesforceHybridSDKTestApp-Info.plist

```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>${PRODUCT_NAME}</string>
+	<string>SalesforceHybridSDKTestApp</string>
```

Comment:
Isn't `${PRODUCT_NAME}` the same as `SalesforceHybridSDKTestApp`? Why hardcode this here?